### PR TITLE
Add version to gRPC service packages

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

Add version to gRPC service packages.

* Changed protobuf messages based on #1409 
* Added comments
* Splitted placement service from daprinternal.proto

Note: I didn't generate go proto stub and client codes. once this pr is merged, I will regen the clients and update the code.

## Issue reference

#1410 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation
* [ ] Specification has been updated
* [ ] Provided sample for the feature

### CODE REVIEW

File: pkg/proto/common/v1/protocol_ext.proto [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+// HTTPExtensions includes HTTP verb and querystring
+// when Dapr runtime delivers HTTP content.
+// 
+// For example, when callers calls http invoke api
+// POST http://localhost:3500/v1.0/invoke/<app_id>/method/<method>?query1=value1&query2=value2
+// 
+// Dapr runtime will parse POST as a verb and extract querystring to quersytring map.
+message HTTPExtensions {
+  // Type of HTTP Verb supported by Dapr.
+  enum HTTPVerb {
```
Comment (amanbha) [inline pkg/proto/common/v1/protocol_ext.proto:24]: can it support all the verbs, there should not be any restrictions from Dapr on it.

Comment (youngbupark) [reply pkg/proto/common/v1/protocol_ext.proto:24]: I refer to the api spec. but yes we can add all verbs. I will add the rest of them.

Comment (amanbha) [reply pkg/proto/common/v1/protocol_ext.proto:24]: Great, thanks @youngbupark

**Pull request merged**